### PR TITLE
Add CentOS 8/Python3 support

### PR DESCRIPTION
**Pull request by bellackn**

### CODE REVIEW

File: tasks/setup_RedHat.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 - name: Install SELinux Python dependencies.
   package:
-    name: libsemanage-python
+    name: "{{ 'libsemanage-python' if ansible_distribution_major_version|int < 8 else 'libsemanage-python3' }}"
```
Comment (tobias-richter) [inline tasks/setup_RedHat.yml]: Please use the "version" comparator instead, see https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison

File: vars/family_RedHat.yml [hunk 2]
```diff
@@ -1,2 +1,6 @@
 aem_dispatcher_module_path: /usr/lib64/httpd/modules
-_aem_dispatcher_apache_server_root: "{{ aem_dispatcher_apache_server_root | default(apache_server_root) | default('/etc/httpd') }}"
\ No newline at end of file
+_aem_dispatcher_apache_server_root: >-
+  {{ aem_dispatcher_apache_server_root | default(apache_server_root) | default('/etc/httpd') }}
+
+aem_dispatcher_libssl_version: >-
+  {{ '1.1.1' if ansible_distribution_major_version|int >= 8 else '1.0.2k' }}
```
Comment (tobias-richter) [inline vars/family_RedHat.yml]: Please use version comparator: https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison

**Review by tobias-richter — CHANGES_REQUESTED** [review]
Thank you! Looks good, only two minor issues.

File: tasks/setup_RedHat.yml (hunk 1 above)
Comment (bellackn) [reply tasks/setup_RedHat.yml]: ✔️

File: vars/family_RedHat.yml (hunk 2 above)
Comment (bellackn) [reply vars/family_RedHat.yml]: ✔️

**Review by tobias-richter — APPROVED** [review]

**Pull request merged**